setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = migrations
+exclude = migrations,.venv
```

Comment:
Is wildcard supported here? e.g. could it be adjusted to also exclude stuff like `.env`, `venv`, `venv_local`, etc (I've seen other people using these names as their local virtual environments.